README.md

```diff
@@ -39,10 +39,12 @@ There are a several reasons to use TPI instead of other related solutions (custo
    TPI is a CLI tool, not a running service. It requires no additional orchestrating machine (control plane/head nodes) to schedule/recover/terminate instances. Instead, TPI runs (spot) instances via cloud-native scaling groups[^scalers], taking care of recovery and termination automatically on the cloud provider's side. This design reduces management overhead & infrastructure costs. You can close your laptop while cloud tasks are running -- auto-recovery happens even if you are offline.
 2. **Unified tool for data science and software development teams**:
    TPI provides consistent tooling for both data scientists and DevOps engineers, improving cross-team collaboration. This simplifies compute management to a single config file, and reduces time to deliver ML models into production.
+3. **Reproducible, codified environments**:
+   Store hardware requirements & pipelines in a single configuration file with the rest of your ML project code.
```

Comment:
```suggestion
   Store hardware requirements in a single configuration file with the rest of your ML project code.
```

As per https://github.com/iterative/terraform-provider-iterative/issues/513#issuecomment-1102383582, the “pipelines” part might be a bit far–fetched.

Reply:
how about this?

```suggestion
   Store hardware requirements in a single configuration file with the rest of your ML pipeline code.
```

Reply:
Sounds better, but still implies (?) that users should store all their pipeline code in a Terraform file. 🙃

Reply:
Hardware requirements (main.tf) are stored with/alongside ML pipeline code (dvc.yaml, train.py) in Git.

Reply:
```suggestion
   Store hardware requirements & pipelines in a single configuration file, alongside with the rest of your ML pipeline code.
```